src/plugins/intel_cpu/tests/functional/shared_tests_instances/single_layer_tests/broadcast.cpp

```diff
@@ -39,24 +39,24 @@ std::vector<std::vector<size_t>> targetShapesNumpy1D = {
 
 const auto numpyBroadcast1DInputParams = ::testing::Combine(
         ::testing::ValuesIn(targetShapesNumpy1D),
-        ::testing::Values(ngraph::AxisSet{}), //not used in numpy mode
-        ::testing::Values(ngraph::op::BroadcastType::NUMPY),
+        ::testing::Values(ov::AxisSet{}), //not used in numpy mode
+        ::testing::Values(ov::op::BroadcastType::NUMPY),
         ::testing::Values(std::vector<size_t>{1}),
         ::testing::ValuesIn(inputPrecisions),
         ::testing::Values(ov::test::utils::DEVICE_CPU)
 );
 
-INSTANTIATE_TEST_CASE_P(smoke_TestNumpyBroadcast1D, BroadcastLayerTest, numpyBroadcast1DInputParams, BroadcastLayerTest::getTestCaseName);
+INSTANTIATE_TEST_CASE_P(smoke_TestNumpyBroadcast1D, BroadcastLayerTestNew, numpyBroadcast1DInputParams, BroadcastLayerTestNew::getTestCaseName);
```

Comment:
I don't believe that you need to introduce the new class. Here you can use `BroadcastLayerTest` name. In case if gtest has any issues with namespaces I think it already should be solved in line 10 by:
```
using namespace ov::test;
```

Reply:
Unfortunately, gtest does not want to work with equal names even if I rename testcases:

`
 Attempted redefinition of test suite BroadcastLayerTest.
All tests in the same test suite must use the same test fixture
class.  However, in test suite BroadcastLayerTest, you tried
to define a test using a fixture class different from the one
used earlier. This can happen if the two fixture classes are
from different namespaces and have the same name. You should
probably rename one of the classes to put the tests into different
test suites.
`

Reply:
Gtest doesn't work if there are 2 or more fixtures with equal name (even in different namespace). I tried to rename testcase or testsuite - doesn't help

`Attempted redefinition of test suite BroadcastLayerTest.
All tests in the same test suite must use the same test fixture
class.  However, in test suite BroadcastLayerTest, you tried
to define a test using a fixture class different from the one
used earlier. This can happen if the two fixture classes are
from different namespaces and have the same name. You should
probably rename one of the classes to put the tests into different
test suites.`

Reply:
We can rename it back after all plugins migrate from old shared_test_classes to new ones